### PR TITLE
- Removes the need for NativeHelpers::Blob

### PR DESCRIPTION
**Pull request by Xliff**
As suggested by @niner, this PR removes the need for NativeHelpers::Blob.

[See this MoarVM issue](https://github.com/MoarVM/MoarVM/issues/1615) for the discussion.

### CODE REVIEW

File: lib/DB/MySQL/Native.pm6 [hunk 1]
```diff
@@ -415,28 +415,26 @@ class DB::MySQL::Native is repr('CPointer')
...
         is native(LIBMYSQL) is symbol('mysql_options') {}
 
     multi method option(mysql-option $option, Str:D $arg)
     {
-        use NativeHelpers::Blob;
         my int32 $o = $option;
 
         $.check(
-          $.mysql_options( $o, pointer-to($arg.encode) )
+          $.mysql_options( $o, CArray[uint8].new($arg.encode) )
```
Comment (niner) [inline lib/DB/MySQL/Native.pm6:428]: I wonder why you can't just do `$.mysql_options($o, $arg.encode)` here? NativeCall does support Blobs: https://github.com/MoarVM/MoarVM/blob/master/src/core/nativecall_dyncall.c#L988
I rely on this support in Inline::Perl5: https://github.com/niner/Inline-Perl5/blob/master/lib/Inline/Perl5.pm6#L62 and https://github.com/niner/Inline-Perl5/blob/master/lib/Inline/Perl5/Interpreter.pm6#L116

**Comment by patrickbkr** [comment]
If this removes all uses of NativeHelpers::Blob, shouldn't the dependency then also be removed from the META6.json?

File: lib/DB/MySQL/Native.pm6 (hunk 1 above)
Comment (Xliff) [reply lib/DB/MySQL/Native.pm6:428]: @niner - Every time I have attempted to use Blobs via NativeCall, I have encountered a SEGV. Given your recommendation, I may take a look at them again. Thanks.

**Comment by Xliff** [comment]
@patrickbkr - Thanks, I will update

**Pull request merged**